draft-ietf-anima-brski-prm.md

```diff
@@ -881,45 +898,45 @@ Optionally, TLS MAY be used to provide privacy for this exchange between the Reg
+The Registrar-Agent triggers the pledge to create a PVR via HTTP POST to `/.well-known/brski/tpvr`.
+The request body MUST contain the JSON-based Pledge Voucher-Request Trigger (tPVR) artifact as defined in {{tpvr-artifact}}.
+In the request header, the Content-Type field MUST be set to `application/json` and the Accept field SHOULD be set to `application/voucher-jws+json` as defined in {{!I-D.ietf-anima-jws-voucher}}.
 
-If the pledge is unable to create the PVR, it SHOULD respond with an HTTP error code. The following client error responses MAY be used:
+Upon receiving a valid tPVR, the pledge MUST reply with the PVR artifact as defined in {{pvr-artifact}} in the body of a 200 OK response.
+In the response header, the Content-Type field MUST be set to `application/voucher-jws+json` as defined in {{!I-D.ietf-anima-jws-voucher}}.
 
-* 400 Bad Request: if the pledge detected an error in the format of the request, e.g. missing field, wrong data types, etc. or if the request is not valid JSON even though the PVR media type was set to `application/json`.
+If the pledge is unable to create the PVR, it SHOULD respond with an HTTP error code.
+The following client error codes MAY be used:
 
-* 406 Not Acceptable: if the Accept request header field indicates a type that is unknown or unsupported, e.g., a type other than `application/jose+json`.
+* 400 Bad Request: if the pledge detects an error in the format of the request, e.g., missing field, wrong data types, etc. or if the request is not valid JSON even though the Content-Type request header field was set to `application/json`
+* 406 Not Acceptable: if the Accept request header field indicates a type that is unknown or unsupported, e.g., a type other than `application/voucher-jws+json`
+* 415 Unsupported Media Type: if the Content-Type request header field indicates a type that is unknown or unsupported, e.g., a type other than `application/json`
 
-* 415 Unsupported Media Type: if the Content-Type request header field indicates a type that is unknown or unsupported, e.g., a type other than `application/json`.
 
-### Request Artifact: Pledge Voucher-Request Trigger (tPVR)
+### Request Artifact: Pledge Voucher-Request Trigger (tPVR) {#tpvr-artifact}
 
-The Pledge Voucher-Request Trigger (tPVR) artifact is an unsigned JSON structure providing the trigger parameters.
-The following CDDL {{!RFC8610}} explains the Pledge Voucher-Request Trigger structure. 
+The Pledge Voucher-Request Trigger (tPVR) artifact SHALL be an unsigned data object, providing the necessary parameters to later assert agent-proximity: domain registrar EE certificate, SubjectKeyIdentifier of the Registrar-Agent EE certificate, and agent-signed product-serial-number of the pledge.
+It is unsigned because at the time of receiving the tPVR, the pledge can verify neither certificate nor signature and can only accept the parameters provisionally until it receives the voucher as described in {{voucher}} (see {{agt_prx}}).
 
```

Comment:
If we state unsigned here, we should underline that it contains a signed artifact (agent-signed-data) that is evaluated by the registrar and also by the MASA. 
Proposal to include the following:
Nevertheless, the tPVR contains a signed artifact, the agent-signed-data, which is included in the PVR and evaluated by the registrar and the MASA.

Reply:
To keep it short (the details how to implement come just below), I would include it in the content list. We wanted to clarify the role of the SubjectKeyIdentifier for the agent-signed-data anyway. Finally, I would not call it an artifact, as it is never stand-alone, but always a field of artifacts. So how about:

>...: the domain registrar EE certificate and an agent-signed data object (containing the product-serial-number and a timestamp), which has to be included in the PVR and whose signature is verified by the registrar and MASA utilizing the more compact SubjectKeyIdentifier of the Registrar-Agent EE certificate.
 The artifact is unsigned because ...

Reply:
Sounds good!